src/foundry/client/data/documents/token.d.mts

```diff
@@ -48,27 +64,33 @@ declare global {
     get inCombat(): boolean;
 
     /**
-     * Define a sort order for this TokenDocument.
-     * This controls its rendering order in the PrimaryCanvasGroup relative to siblings at the same elevation.
-     * In the future this will be replaced with a persisted database field for permanent adjustment of token stacking.
-     * In case of ties, Tokens will be sorted above other types of objects.
+     * The Regions this Token is currently in.
+     * @privateRemarks TODO: Change Set<any> to Set<RegionDocuments> when RegionDocuments are implemented
      */
-    get sort(): number;
-
-    set sort(value);
+    regions: Set<any> | null;
```

Comment:
Foundry documents this as a `Set<RegionDocument>` and looking into it I'm inclined to believe them. Even if this isn't set up yet please leave at minimum TODO or better yet a currently not resolving import to where `RegionDocument` should be.